staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -653,6 +660,12 @@ func tryUnmount(path string, unmountTimeout time.Duration) error {
 	}
 
 	if cmderr != nil {
+		if strings.Contains(string(out), "not mounted") {
```

Comment:
I think you need to read stderr and not stdout here.

```
umount abc > stdout.txt 2> stderr.txt
cat stderr.txt 
umount: abc: No such file or directory
 cat stdout.txt 
```

Reply:
it still works because we are using CombinedOutput which combines stderr and stdout